theano/link/basic.py

```diff
@@ -323,16 +351,25 @@ def accept(self, fgraph, no_recycling=None, profile=None):
         self.no_recycling = no_recycling
         return self
 
-    def make_all(self, input_storage=None, output_storage=None, storage_map=None):
+    def make_all(
+        self,
+        input_storage: Optional[Any] = None,
+        output_storage: Optional[Any] = None,
+        storage_map: Optional[Any] = None,
```

Comment:
These are not specific enough

Reply:
See the link in [this comment](https://github.com/pymc-devs/Theano-PyMC/pull/272#discussion_r564907981) for more specific type information regarding this case.

Reply:
Now pointing to the new type variables:

```python
self,
input_storage: Optional[Any] = None,
output_storage: Optional[OutputStorageType] = None,
storage_map: Optional[StorageMapType] = None,
```